soco/core.py

```diff
@@ -2153,6 +2158,70 @@ def get_sonos_playlist_by_attr(self, attr_name, match):
                 return sonos_playlist
         raise ValueError('No match on "{0}" for value "{1}"'.format(attr_name, match))
 
+    def get_battery_info(self, timeout=3.0):
+        """Get battery information for a Sonos speaker.
+
+        Obtains battery information for Sonos speakers that report it. This only
+        applies to Sonos Move speakers at the time of writing.
+
+        This method may only work on Sonos 'S2' systems.
+
+        Uses the 'support/review' URL, which collects comprehensive
+        system information from all players in the system via the target device,
+        so it's a somewhat expensive call and should be used sparingly.
+
+        Args:
+            timeout (float, optional): The timeout to use when making the
+                HTTP request.
+
+        Returns:
+            dict: A `dict` containing battery status data.
+
+            Example return value::
+
+                {'Health': 'GREEN',
+                 'Level': '100',
```

Comment:
Should we maybe parse the Level and make it an int?

Reply:
I did consider reaching into the battery data, but concluded that it's less brittle just to gather what's there and pass it back without making any assumptions.

Reply:
OK, if "Level" is present in the dict, it now gets converted to an int. Will still work if the fields change.